docs/setup.md

```diff
@@ -22,11 +22,118 @@ Threat Intel Management (TIM) Solution Pack requires the following solution pack
 
 For optimal performance of **Threat Intel Management** solution pack, you can install and configure:
 
-- The MITRE ATT&CK connector with its ingestion set up to ensure ingestion of ATT&CK data in your threat mitigation environment. We recommend that you schedule the data ingestion for at least once a month.
-  - To configure and use the MITRE ATT&CK connector as a source of Mitre Att&ck information, refer to [Configuring Mitre Att&ck](https://docs.fortinet.com/fortisoar/connectors/mitreattack)
-
 - The FortiRecon ACI connector with its ingestion set up to ensure ingestion of FortiRecon ACI data in threat mitigation environment.
-  - To configure and use the FortiRecon ACI connector as a source of Adversary Centric Intelligence(ACI) information, refer to [Configuring FortiRecon ACI](https://docs.fortinet.com/fortisoar/connectors/fortirecon-aci)
+	- To configure and use the FortiRecon ACI connector as a source of Adversary Centric Intelligence(ACI) information, refer to [Configuring FortiRecon ACI](https://docs.fortinet.com/fortisoar/connectors/fortirecon-aci).
+
+- The Exchange connector with email ingestion setup to ensure regular ingestion of threat feeds from email attachments.
+	- To configure and use the Exchange connector, refer to [Configuring Exchange](https://docs.fortinet.com/fortisoar/connectors/exchange).
+
+## Setup Threat Intel Management on FortiSOAR
+
+The option to launch the Threat Intel Management Configuration Wizard appears on the Threat Intel Management solution pack's content hub page.
```

Comment:
Please include landing page with its SS and explain it here as well - since its another option to configure TIM (for the 1st time)

Reply:
Done